docs/src/main/paradox/googlepubsub.md

```diff
@@ -0,0 +1,64 @@
+# Google Pub Sub
+
+The google pub sub connector provides a way to connect to google clouds managed pub sub https://cloud.google.com/pubsub/. A global service for real-time and reliable messaging and streaming data.
+
+## Artifacts
+
+sbt
+:   @@@vars
+    ```scala
+    libraryDependencies += "com.lightbend.akka" %% "akka-stream-alpakka-google-pubsub" % "$version$"
+    ```
+    @@@
+
+Maven
+:   @@@vars
+    ```xml
+    <dependency>
+      <groupId>com.lightbend.akka</groupId>
+      <artifactId>akka-stream-alpakka-google-pubsub_$scala.binaryVersion$</artifactId>
+      <version>$version$</version>
+    </dependency>
+    ```
+    @@@
+
+Gradle
+:   @@@vars
+    ```gradle
+    dependencies {
+      compile group: "com.lightbend.akka", name: "akka-stream-alpakka-google-pubsub_$scala.binaryVersion$", version: "$version$"
+    }
+    ```
+    @@@
+                                                                                                                          
+## Usage
+
+Prepare your credentials for access to google pub sub.
+
+Scala
+: @@snip (../../../../googlepubsub/src/test/scala/akka/stream/alpakka/googlepubsub/ExampleUsage.scala) { #init-credentials }
+
+And prepare the actor system and materializer.
+
+Scala
+: @@snip (../../../../googlepubsub/src/test/scala/akka/stream/alpakka/googlepubsub/ExampleUsage.scala) { #init-mat }
+
+To publish a single request, build the message with a base64 data payload and put it in a PublishRequest. Publishing creates a flow taking the messages and returning the accepted message ids.
```

Comment:
Done, Do I need to actually add a scaladoc, or will it link to the source or something ?